packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.test.tsx

```diff
@@ -2,22 +2,34 @@ import 'whatwg-fetch';
 import { h } from 'preact';
 import { v4 as uuidv4 } from 'uuid';
 import { render } from '@testing-library/preact';
+import userEvent from '@testing-library/user-event';
+
+import { ThemeProvider } from '../../../providers';
 
 import { Autocomplete } from '../../Organisms/Autocomplete/Autocomplete';
-import { createAutocompleteController } from '@searchspring/snap-preact';
```

Comment:
Any particular reason to drop the use of `createAutocompleteController`? This test got a lot more complicated without it... You can pass services to the function in order to utilize the MockClient. For example:
```
const controller = createAutocompleteController({ client, controller: config }, {client: mockClient });
```